### PR TITLE
Locales: Swedish and Norwegian

### PR DESCRIPTION
**Pull request by melloware**
Requested from PrimeFaces ticket: https://github.com/primefaces/primefaces/issues/6932

### CODE REVIEW

**Comment by melloware** [comment]
Any way we can get this merged and a release cut?

File: docs/timeline/index.html [hunk 1]
```diff
@@ -2239,10 +2239,26 @@ <h3 id="available-locales">Available locales</h3>
       <td>Portuguese</td>
       <td>
         <code>pt</code><br>
-        <code>pt_BR</code>
+        <code>pt_BR</code><br>
```
Comment (mojoaxel) [inline docs/timeline/index.html:2242]: This merge request should only touch the Swedish and Norwegian parts and leave the other code alone for not. It Portuguese need change this should be done in a seperate merge request.

**Review by mojoaxel — CHANGES_REQUESTED** [review]

**Comment by mojoaxel** [comment]
@melloware Thanks for this pull-request! Let's merge that soon!

**Review by mojoaxel — APPROVED** [review]
LGTM!

**Review by yotamberk — APPROVED** [review]

**Pull request merged**